app/models/ride.rb

```diff
@@ -8,10 +8,37 @@ class Ride < ApplicationRecord
   belongs_to :ride_zone
   has_one :conversation
 
+  scope :completed, -> { where(status: :complete)}
+
   validates :voter, presence: true
 
   after_create :notify_creation
   around_save :notify_update
+  before_save :close_conversation
+
+  include HasAddress
+
+  # create a new ride from the data in a conversation
+  def self.create_from_conversation conversation
+    attrs = {
+      voter_id: conversation.user_id,
+      name: conversation.username,
+      pickup_at: conversation.pickup_time,
+      status: :scheduled,
```

Comment:
You're right, I was making an assumption that only when the conversation was complete would a ride be created from it. But at this point, I don't think we have a use case for the creation of a Ride object unless it is scheduled. So we probably want to remove the :incomplete_info status